src/test/java/uk/gov/justice/digital/delius/integration/secure/CustodyUpdateNomsNumberAPITest.java

```diff
@@ -0,0 +1,123 @@
+package uk.gov.justice.digital.delius.integration.secure;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.applicationinsights.TelemetryClient;
+import io.restassured.RestAssured;
+import io.restassured.config.ObjectMapperConfig;
+import io.restassured.config.RestAssuredConfig;
+import org.flywaydb.core.Flyway;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.jdbc.core.ColumnMapRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.justice.digital.delius.data.api.IDs;
+import uk.gov.justice.digital.delius.data.api.UpdateOffenderNomsNumber;
+import uk.gov.justice.digital.delius.jwt.JwtAuthenticationHelper;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ExtendWith(SpringExtension.class)
+@ActiveProfiles("dev-seed")
+@DirtiesContext
+@Disabled("Disabled until CI memory issues are addressed")
```

Comment:
Sadly these are still disabled until the H2 out of memory issue is fixed (on PiCT team backlog)

Reply:
Is it worth sticking a TODO on there pointing at the PiCT JIRA ticket?

Reply:
Good idea